subscription-manager.spec

```
@@ -1325,6 +1331,9 @@ find %{buildroot} -name \*.py* -exec touch -r %{SOURCE0} '{}' \;
     %endif
 %endif
 
+# Make all entitlement certificates and keys files readable by group and other
+chmod go+r /etc/pki/entitlement/*.pem
```

We need this not to fail on error. I'd suggest the following:
```suggestion
chmod go+r /etc/pki/entitlement/*.pem | true
```

---

What you probably want to use here is `chmod go+r /etc/pki/entitlement/*.pem || true`, because single `|` works here as unnamed pipe. The result will be the same in this case.

---

Why it could be important? Try this in console:
```
echo "hello" | true
```
and then try this:
```
echo "hello" || true
```
Do you see it? The output from the first case is consumed by `true` command and it is not printed to stdout.

---

But you are true that we don't want to fail on error. :-) Force pushed new update.